addons/mail/static/src/scss/variables/primary_variables.scss

```diff
@@ -11,3 +11,6 @@ $o-mail-Message-sidebarWidth: 48px !default;
 $o-mail-LinkPreview-width: 320px !default;
 $o-mail-LinkPreview-height: 240px !default;
 $o-mail-LinkPreviewCard-height: 80px !default;
+$o-mail-CallMenu-bgColor-darker: #111827FF !default;
+$o-mail-CallMenu-bgColor: #1F2937FF !default;
+$o-mail-CallMenu-bgColor-lighter: #374151FF !default;
```

Comment:
I think `scss/variables/primary_variables.scss` is deprecated SCSS file. We should use `core/common/primary_variables.scss` instead.